src/Javis.jl

```diff
@@ -1015,6 +1020,7 @@ function javis(
     actions::Vector{AA};
     framerate = 30,
     pathname = "javis_$(randstring(7)).gif",
+    liveview = false,
```

Comment:
this doesn't seem to be documented in the docstring

Reply:
Added to the docstring. Feel free to resolve the convo